x-pack/test/functional/apps/api_keys/home_page.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export default ({ getPageObjects, getService }: FtrProviderContext) => {
+  const pageObjects = getPageObjects(['common', 'apiKeys']);
+  const log = getService('log');
+
+  describe('Home page', function() {
+    this.tags('smoke');
+    before(async () => {
+      await pageObjects.common.navigateToApp('apiKeys');
+    });
+
+    it('Loads the app', async () => {
+      await log.debug('Checking for section header');
```

Comment:
nit: `log.debug` is not an async function ([source](https://github.com/elastic/kibana/blob/7739f5da9ed682d69cfbb1dafd5192c006339db2/packages/kbn-dev-utils/src/tooling_log/tooling_log.ts#L45-L47))
```suggestion
     log.debug('Checking for section header');
```